assets/svg/icons.svg

```diff
@@ -595,5 +595,27 @@
+	</g>
+  </symbol>
+
+  <symbol id="icon-call" viewBox="0 0 247.383 247.564">
+    <path d="M1373.837,1347.757a11.357,11.357,0,0,0-9.618-11.726,156.528,156.528,0,0,1-34.255-8.524,11.281,11.281,0,0,0-11.878,2.5l-14.355,14.337a11.308,11.308,0,0,1-13.575,1.827,191.971,191.971,0,0,1-72.052-71.975,11.268,11.268,0,0,1,1.833-13.56l14.31-14.295a11.3,11.3,0,0,0,2.536-11.937,155.4,155.4,0,0,1-8.517-34.094,11.326,11.326,0,0,0-11.412-9.708h-33.9a11.317,11.317,0,0,0-8.337,3.676,11.2,11.2,0,0,0-2.924,8.454,212.026,212.026,0,0,0,33,93.022,209.112,209.112,0,0,0,64.424,64.332,212.364,212.364,0,0,0,92.4,32.888,11.306,11.306,0,0,0,12.321-11.341Zm22.6.126v33.7a33.893,33.893,0,0,1-37.161,33.849A235.032,235.032,0,0,1,1256.9,1379.08a231.52,231.52,0,0,1-71.233-71.136,234.692,234.692,0,0,1-36.466-103.007,33.885,33.885,0,0,1,33.746-36.922h33.8a33.917,33.917,0,0,1,33.916,29.234,133.384,133.384,0,0,0,7.288,29.227,33.869,33.869,0,0,1-7.676,35.78l-8.1,8.1A169.456,169.456,0,0,0,1294,1322.13l8.15-8.146a33.919,33.919,0,0,1,35.743-7.622,134.193,134.193,0,0,0,29.385,7.3A33.882,33.882,0,0,1,1396.44,1347.883Z" transform="translate(-1149.064 -1168.013)" />
+  </symbol>
+
+  <symbol id="icon-email-outline" viewBox="0 0 301.5 245.77">
+  	<path d="M41.12,245.77A41.106,41.106,0,0,1,.009,204.813V41.426c-.014-.36-.012-.721.005-1.082A41.122,41.122,0,0,1,41.12,0h219.26a41.125,41.125,0,0,1,41.108,40.349c.016.358.017.717,0,1.074v163.39a41.107,41.107,0,0,1-41.112,40.957ZM27.413,204.813A13.764,13.764,0,0,0,41.12,218.465h219.26a13.765,13.765,0,0,0,13.708-13.653V67.194l-115.48,80.539a13.776,13.776,0,0,1-15.715,0L27.413,67.194ZM150.75,119.875l121.876-85a13.792,13.792,0,0,0-12.247-7.569H41.12a13.792,13.792,0,0,0-12.245,7.569Z" />
+  </symbol>
+
+  <symbol id="icon-livechat" viewBox="0 0 249.853 249.863">
```

Comment:
<img width="280" alt="Screenshot 2022-10-24 at 11 27 22" src="https://user-images.githubusercontent.com/9892048/197505907-3c3d5d16-d596-49e2-aa7e-08396575182c.png">

Should there be a grey background?

Reply:
Yes, Tom provided the icon with the grey background